src/payment/strategies/offsite/offsite-payment-strategy.ts

```diff
@@ -32,7 +33,8 @@ export default class OffsitePaymentStrategy implements PaymentStrategy {
                     payment.methodId,
                     payment.gatewayId,
                     instrumentId,
-                    shouldSaveInstrument))
+                    shouldSaveInstrument,
+                    getBrowserInfo()))
```

Comment:
That should go in a new line:

```suggestion
                    getBrowserInfo()
                )));
```